napi CLI doesn't report version correctly

```
❯ napi version
Internal Error: Cannot find module 'C:\Users\kiese\package.json'
Require stack:
- C:\Users\kiese\AppData\Roaming\npm\node_modules\@napi-rs\cli\scripts\index.js
Require stack:
- C:\Users\kiese\AppData\Roaming\npm\node_modules\@napi-rs\cli\scripts\index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at getNapiConfig (C:\Users\kiese\AppData\Roaming\npm\node_modules\@napi-rs\cli\scripts\index.js:8281:21)
    at VersionCommand.<anonymous> (C:\Users\kiese\AppData\Roaming\npm\node_modules\@napi-rs\cli\scripts\index.js:168258:44)
    at Generator.next (<anonymous>)
    at C:\Users\kiese\AppData\Roaming\npm\node_modules\@napi-rs\cli\scripts\index.js:3467:71
    at new Promise (<anonymous>)
    at __awaiter$1 (C:\Users\kiese\AppData\Roaming\npm\node_modules\@napi-rs\cli\scripts\index.js:3463:12)
```

--- a/cli/src/update-package.ts
+++ b/cli/src/update-package.ts
@@ -1,9 +1,17 @@
-import { writeFileAsync } from './utils'
+import { debugFactory } from './debug'
+import { writeFileAsync, fileExists } from './utils'
+
+const debug = debugFactory('update-package')
 
 export async function updatePackageJson(
   path: string,
   partial: Record<string, any>,
 ) {
+  const exists = await fileExists(path)
+  if (!exists) {
+    debug(`File not exists ${path}`)
+    return
+  }
   const old = require(path)
   await writeFileAsync(path, JSON.stringify({ ...old, ...partial }, null, 2))
 }

--- a/cli/src/utils.ts
+++ b/cli/src/utils.ts
@@ -1,4 +1,4 @@
-import { readFile, writeFile, copyFile, mkdir, unlink } from 'fs'
+import { readFile, writeFile, copyFile, mkdir, unlink, stat } from 'fs'
 import { promisify } from 'util'
 
 export const readFileAsync = promisify(readFile)
@@ -6,6 +6,14 @@ export const writeFileAsync = promisify(writeFile)
 export const unlinkAsync = promisify(unlink)
 export const copyFileAsync = promisify(copyFile)
 export const mkdirAsync = promisify(mkdir)
+export const statAsync = promisify(stat)
+
+export async function fileExists(path: string) {
+  const exists = await statAsync(path)
+    .then(() => true)
+    .catch(() => false)
+  return exists
+}
 
 export function pick<O, K extends keyof O>(o: O, ...keys: K[]): Pick<O, K> {
   return keys.reduce((acc, key) => {

